aplib.net-demo/Assets/Scripts/Entities/Weapons/MeleeWeapon.cs

```diff
@@ -13,19 +13,22 @@ public class MeleeWeapon : Weapon
         /// <summary>
         /// The amount of damage the weapon deals.
         /// </summary>
-        [SerializeField] public int Damage = 25;
+        [SerializeField]
+        public int Damage = 25;
```

Comment:
Is it correct that we have a Damage field and a _damage field now? Either way, that should probably change. Also, quick tip for later: you can make a public auto-property serialized in the inspector like so:
```suggestion
        [field: SerializeField]
        public int Damage { get; set; } = 25;
```